src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="System.Security.AccessControl" Version="4.4.0-preview1-25302-01" />
     <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.4.0-preview1-25302-01" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="4.4.0-preview1-25302-01" />
+    <PackageReference Include="System.Threading.AccessControl" Version="4.4.0-preview1-25302-01" />
```

Comment:
this is required by types moved from System.Security.AccessControl to System.Threading.AccessControl (System.Security.AccessControl.MutexRights), am I allowed to add new references here ?